synapse/config/logger.py

```diff
@@ -68,9 +68,6 @@
         filters: [context]
 
 loggers:
-    synapse:
-        level: INFO
-
```

Comment:
Why remove this and not the `synapse.storage.SQL` below? It feels nice to give a place to easily up the synapse log level?

Reply:
Do you think it's particularly likely that we would need to up the synapse log level and not other loggers? ie, why wouldn't you do this in the `root` logger?

The separate config for synapse.storage.SQL exists so that if you set your logging to DEBUG (either at the root level, or just for `synapse`), you don't start logging all the access tokens etc that go in and out of the database.